src/test/java/com/epam/ta/reportportal/ws/converter/builders/TestItemBuilderTest.java

```diff
@@ -0,0 +1,109 @@
+package com.epam.ta.reportportal.ws.converter.builders;
+
+import com.epam.ta.reportportal.entity.ItemAttribute;
+import com.epam.ta.reportportal.entity.enums.StatusEnum;
+import com.epam.ta.reportportal.entity.enums.TestItemTypeEnum;
+import com.epam.ta.reportportal.entity.item.Parameter;
+import com.epam.ta.reportportal.entity.item.TestItem;
+import com.epam.ta.reportportal.entity.item.TestItemResults;
+import com.epam.ta.reportportal.entity.launch.Launch;
+import com.epam.ta.reportportal.ws.model.ItemAttributeResource;
+import com.epam.ta.reportportal.ws.model.ParameterResource;
+import com.epam.ta.reportportal.ws.model.StartTestItemRQ;
+import com.google.common.collect.Sets;
+import org.junit.Test;
+
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.util.Collections;
+import java.util.Date;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.*;
+
+/**
+ * @author <a href="mailto:ihar_kahadouski@epam.com">Ihar Kahadouski</a>
+ */
+public class TestItemBuilderTest {
+
+	@Test
+	public void testItemBuilder() {
+		final Launch launch = new Launch();
+		launch.setId(1L);
+		launch.setName("name");
+		final ParameterResource parameterResource = new ParameterResource();
+		parameterResource.setKey("key");
+		parameterResource.setValue("value");
+		final String description = "description";
+		final String typeValue = "step";
+		final TestItem parent = new TestItem();
+		final TestItem testItem = new TestItemBuilder().addDescription(description)
+				.addType(typeValue)
+				.addLaunch(launch)
+				.addParameters(Collections.singletonList(parameterResource))
+				.addAttributes(Sets.newHashSet(new ItemAttributeResource("key", "value")))
+				.addParent(parent)
+				.get();
+
+		assertThat(testItem.getLaunch()).isEqualToComparingFieldByField(launch);
+		assertEquals(description, testItem.getDescription());
+		assertEquals(TestItemTypeEnum.STEP, testItem.getType());
+		final Parameter param = new Parameter();
+		param.setKey("key");
+		param.setValue("value");
+		assertTrue(testItem.getParameters().contains(param));
+		assertThat(testItem.getAttributes()).containsExactly(new ItemAttribute("key", "value", false));
+		assertNotNull(testItem.getParent());
+	}
+
+	@Test
+	public void startRq() {
```

Comment:
more clarified names for tests please
test name should provide information about what it is testing